http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/KubernetesAppConfig.scala

```diff
@@ -34,6 +23,15 @@ sealed trait KubernetesAppConfig {
   def kubernetesServices: List[KubernetesService]
 
   def enabled: Boolean
+
+  // Each app can configure its own list of chart versions to NEVER update. To be used
+  // when changes aren't backward-compatible and we know an update would be destructive.
+  def chartVersionsToExcludeFromUpdates: List[ChartVersion]
+
+  // These are defined by each implementing class. Each config type
+  // corresponds to a specific app type and cloud provider.
+  def cloudProvider: CloudProvider
```

Comment:
I was wondering if app configs should be cloud specific… but I guess they are today, I guess due to differences in the charts/services.

Reply:
They are all cloud specific for now yes